cosi/protocol/cosi.go

```diff
@@ -221,7 +221,7 @@ func (c *CoSi) handleCommitment(in *Commitment) error {
 	}
 
 	// go to Commit()
-	out := c.cosi.Commit(nil, c.tempCommitment)
+	out := c.cosi.Commit(c.Suite().RandomStream(), c.tempCommitment)
```

Comment:
Question: Why not `c.Suite()` instead of `c.Suite().RandomStream()`?

Reply:
When we switch to kyber/sign/cosi, it will be, because there the signature of Commit is:
`func Commit(suite Suite) (kyber.Scalar, kyber.Point)`